plugins/node.d/smart_

```diff
@@ -589,6 +589,23 @@ def find_smart_drives():
                     continue
         except Exception as exc:
             verboselog('Failed to list FreeBSD disks: {}'.format(exc))
+    elif os.uname()[0] == "MidnightBSD":
```

Comment:
After taking another look at this: I think, you can avoid code duplication here by simply changing the above condition from `elif os.uname()[0] == "NetBSD"` to `elif os.uname()[0] in {"NetBSD", "MidnightBSD"}:`.

And maybe customize the final error message as `verboselog('Failed to list {} disks: {}'.format(os.uname()[0], exc))`